### PR TITLE
Add back normalization on FS_EVENT removed when migrating to ProtoConcatenator concatToMap function

### PR DESCRIPTION
**Pull request by ashangit**
Due to this bug uri can be different for same nameservice access and path can contain hdfs namenservice

### CODE REVIEW

File: readers/elasticsearch/src/main/java/com/criteo/hadoop/garmadon/elasticsearch/ElasticSearchReader.java [hunk 1]
```diff
@@ -111,6 +111,15 @@ void writeToES(GarmadonMessage msg) {
         } else {
             Map<String, Object> eventMap = ProtoConcatenator.concatToMap(Arrays.asList(msg.getHeader(), msg.getBody()), true);
             eventMap.put("event_type", msgType);
+
+            // Specific normalization for FS_EVENT
+            if ("FS_EVENT".equals(msgType)) {
+                String uri = (String) eventMap.get("uri");
+                eventMap.computeIfPresent("src_path", (k, v) -> ((String) v).replace(uri, ""));
```
Comment (gray-matter) [inline readers/elasticsearch/src/main/java/com/criteo/hadoop/garmadon/elasticsearch/ElasticSearchReader.java]: I'm guessing we disregard cases where uri might be in the middle of these strings (just checking)

File: readers/elasticsearch/src/main/java/com/criteo/hadoop/garmadon/elasticsearch/ElasticSearchReader.java [hunk 2]
```diff
@@ -111,6 +111,15 @@ void writeToES(GarmadonMessage msg) {
...
+                eventMap.computeIfPresent("dst_path", (k, v) -> ((String) v).replace(uri, ""));
+                eventMap.computeIfPresent("uri", (k, v) -> UriHelper.getUniformizedUri((String) v));
```
Comment (gray-matter) [inline readers/elasticsearch/src/main/java/com/criteo/hadoop/garmadon/elasticsearch/ElasticSearchReader.java]: This means that no matter where the file is coming from, we consider its prefix to be that of the current cluster? If we start copying files between different clusters, wouldn't that be an issue?

File: readers/elasticsearch/src/main/java/com/criteo/hadoop/garmadon/elasticsearch/ElasticSearchReader.java (hunk 1 above)
Comment (ashangit) [reply readers/elasticsearch/src/main/java/com/criteo/hadoop/garmadon/elasticsearch/ElasticSearchReader.java]: "hdfs://<nameservice>" or "viewfs://<namservice" should not, I think we can disregard

File: readers/elasticsearch/src/main/java/com/criteo/hadoop/garmadon/elasticsearch/ElasticSearchReader.java [hunk 3]
```diff
@@ -111,6 +111,15 @@ void writeToES(GarmadonMessage msg) {
         } else {
             Map<String, Object> eventMap = ProtoConcatenator.concatToMap(Arrays.asList(msg.getHeader(), msg.getBody()), true);
             eventMap.put("event_type", msgType);
+
+            // Specific normalization for FS_EVENT
+            if ("FS_EVENT".equals(msgType)) {
```
[the remark this answers is not in the document]
Comment (ashangit) [reply readers/elasticsearch/src/main/java/com/criteo/hadoop/garmadon/elasticsearch/ElasticSearchReader.java]: ?

File: readers/elasticsearch/src/main/java/com/criteo/hadoop/garmadon/elasticsearch/ElasticSearchReader.java (hunk 2 above)
Comment (gray-matter) [reply readers/elasticsearch/src/main/java/com/criteo/hadoop/garmadon/elasticsearch/ElasticSearchReader.java]: OK, sorry I misread the code. This is more like a "base URI"/"namespace URI" than an actual full URI

File: readers/elasticsearch/src/main/java/com/criteo/hadoop/garmadon/elasticsearch/ElasticSearchReader.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (gray-matter) [reply readers/elasticsearch/src/main/java/com/criteo/hadoop/garmadon/elasticsearch/ElasticSearchReader.java]: Some constant holding "FS_EVENT". Just curious whether this is used somewhere else, in which case a constant could be useful (but not mandatory).

**Pull request merged**